### PR TITLE
Housekeeping

### PR DESCRIPTION
**Pull request by mkrizek**

### CODE REVIEW

File: migrate.py [hunk 1]
```diff
@@ -183,6 +126,16 @@ def get_plugin_collection(plugin_name, plugin_type, spec):
     raise LookupError('Could not find "%s" named "%s" in any collection in the spec' % (plugin_type, plugin_name))
 
 
+def get_plugins_from_collection(collection, plugin_type, spec):
+    assert collection in spec
+    return [plugin.rsplit('/')[-1][:-3] for plugin in spec[collection].get(plugin_type, [])]
+
+
+def get_plugin_fqcn(namespace, collection, plugin_name):
```
Comment (webknjaz) [inline migrate.py]: This is not needed anymore.

File: migrate.py [hunk 2]
```diff
@@ -565,118 +482,118 @@ def mark_moved_resources(checkout_dir, collection, migrated_to_collection):
...
-
-                    # fix import_role calls?
-                    #tasks = yaml.load(ydata)
-                    #import epdb; epdb.st()
-
-                    if ydata != _ydata:
-                        logger.info('fixing module calls in %s' % yf)
-                        with open(yf, 'w') as f:
-                            f.write(ydata)
+#def copy_tests(plugin, coll, spec, args):
```
Comment (webknjaz) [inline migrate.py]: I think since it's not called, there's no need to comment this out.

File: migrate.py (hunk 1 above)
Comment (mkrizek) [reply migrate.py]: This will be needed.

File: migrate.py (hunk 2 above)
Comment (mkrizek) [reply migrate.py]: MIght be a guide for when we write the functionality. Once we rewrite it, we can remove it.

File: migrate.py [hunk 3]
```diff
@@ -183,6 +126,16 @@ def get_plugin_collection(plugin_name, plugin_type, spec):
     raise LookupError('Could not find "%s" named "%s" in any collection in the spec' % (plugin_type, plugin_name))
 
 
+def get_plugins_from_collection(collection, plugin_type, spec):
+    assert collection in spec
+    return [plugin.rsplit('/')[-1][:-3] for plugin in spec[collection].get(plugin_type, [])]
+
+
+def get_plugin_fqcn(namespace, collection, plugin_name):
+    return '%s.%s.%s' % (namespace, collection, plugin_name)
```
Comment (webknjaz) [inline migrate.py:153]: So maybe 
```suggestion
    return f'{namespace}.{collection}.{plugin_name}'
```
for readability?

File: migrate.py (hunk 2 above)
Comment (webknjaz) [reply migrate.py]: Yes. Just don't comment it out. It's more useful in its normal state as editors will highlight it.

Comment (webknjaz) [reply migrate.py]: thanks

**Review by webknjaz — APPROVED** [review]

**Pull request merged**